mappings/net/minecraft/util/math/Range.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS net/minecraft/unmapped/C_gbowqdyu net/minecraft/util/math/Range
+	COMMENT Represents a range between two comparable objects from a minimum inclusive to a maximum inclusive.
+	FIELD f_itmrigur min Ljava/lang/Comparable;
+	FIELD f_ptgbnwhf INT_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_srmndtze max Ljava/lang/Comparable;
+	METHOD <init> (Ljava/lang/Comparable;Ljava/lang/Comparable;)V
+		ARG 1 min
+		ARG 2 max
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 other
```

Comment:
Yarn has the convention to name these parameters `o` (and Quilt Mappings seems to use those as well).

Reply:
damn it, forgot about it